### DISCUSSION TITLE
Sonoff Dual R3 multi-purpose setup

### DISCUSSION THREAD

**Discussion by vsterian** [arendst/Tasmota#21750] [Q&A]
Hey guys!

I have a scenario where I'm trying to figure out if there is any possiblity to use a tasmotized Sonoff Dual R3 for two purposes at once.

Post installing tasmota on the hardware, I have been using it to control a shutter , which works as expected as configured in https://tasmota.github.io/docs/Blinds-and-Shutters/ 

Now, I want to use the same sonoff device to measure the consumption of an air conditioner that I have installed in the same room. 
The question would be if this scenario is possible at all, like keeping the same shutter config and use the s1/s2 inputs from sonoff dual r3 to measure the energy used (at least) . I'm not necessarily interested in having a relay set in place to turn on/off the ac system.

Any thoughts?

Thanks,
Vlad

**Comment by sfromis** [comment]
As S1/S2 are for mains ac power input signals no way of having sensors connected there.

**Reply by vsterian** [reply]
So the quick answer would be no, it's not possible. Thank you !
